neo4j-rake_tasks.gemspec

```diff
@@ -25,26 +25,27 @@ DESCRIPTION
 
   s.require_path = 'lib'
   s.files = Dir.glob('{bin,lib,config}/**/*') +
-    %w(README.md Gemfile neo4j-rake_tasks.gemspec)
+            %w[README.md Gemfile neo4j-rake_tasks.gemspec]
   s.has_rdoc = true
-  s.extra_rdoc_files = %w( README.md )
+  s.extra_rdoc_files = %w[README.md]
   s.rdoc_options = [
     '--quiet',
     '--title',
     '--line-numbers',
     '--main',
     'README.rdoc',
-    '--inline-source']
+    '--inline-source'
+  ]
 
-  s.add_dependency('rake')
   s.add_dependency('os')
+  s.add_dependency('rake')
   s.add_dependency('ruby-progressbar')
-  s.add_dependency('rubyzip', '>= 1.1.7')
+  s.add_dependency('rubyzip', '~> 1.2', '>= 1.2.1')
```

Comment:
So I specifically needed because `axlsx 2.x` uses `rubyzip 1.0.0`  and `axlsx 3.0.0.pre` uses `rubyzip >=1.2.1`

With some further digging, I learned that there was a vulnerability in `rubyzip 1.1.x` as well - see https://github.com/rubyzip/rubyzip/issues/315